orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -1,47 +1,36 @@
 #!/bin/bash
 # backup_db.sh
 # This script uses the 'pg_dump' utility to dump the contents of hive and ranger PostgreSQL databases as plain SQL.
-# After PostgreSQL contents are dumped, the SQL is uploaded to AWS or Azure using their CLI clients, 'aws s3 cp' and 'azcopy copy' respectively.
-# For the Azure upload, we must write the SQL commands to a local disk before running the `azcopy copy` command.
-# For AWS, the cli supports piping file contents to stdin, so we do that and avoid writing to a local file.
+# After PostgreSQL contents are dumped, the SQL file is uploaded to an HDFS URL using the hdfs cli.
```

Comment:
URL is not an HDFS URL . You can just say " .... the SQL file is uploaded to the cloud storage using the hdfs cli."

Reply:
Done